frontend/src/components/Header/Skiplinks.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Skiplinks as DSSkiplinks, SkiplinkItem } from '@dataesr/react-dsfr';
+
+const Skiplinks = () => (
+  <DSSkiplinks>
+    <SkiplinkItem href="#contenu">Contenu</SkiplinkItem>
+    <SkiplinkItem href="#header-navigation">Menu</SkiplinkItem>
```

Comment:
I don't think we have a header-navigation ?

Reply:
Yes, it's done by react-dsfr